### DISCUSSION TITLE
Import/ allow fixed custom shortcuts instead of the most visited

### DISCUSSION THREAD

**Discussion by MoathDawahreh** [dailydotdev/daily#379] [Feature Requests]
Instead of the most visited, I would love to see the quick access websites instead. If this feature is available I would keep this extension FOREVER

**Comment by nimrodkra** [comment]
We’re now building this feature. You’ll be able to choose between having custom shortcuts (that you curate) or the most visited sites. It should be out in the coming few weeks 🚀

**Comment by rebelchris** [comment]
Done!
